content/news/2023-02-20-global-team-recruiting/index.en.md

```diff
@@ -0,0 +1,104 @@
+---
+title: "Global Team Recruiting"
+author: "Global Leadership Team"
+date: "2023-02-20"
+description: ""
+tags: 
+- global
+- team
+categories:
+- r-ladies
+---
+
+# TL; DR
+
+The R-Ladies Global Team is recruiting new members!
+Please fill [this form](https://docs.google.com/forms/d/1OZB6eVrsocvRyqD-ec5ui69SfXuOR8AZ_L-33B2d5yM/edit?pli=1) to submit yourself for consideration.
+
+# Background
+
+The R-Ladies Global Team is made of volunteers that dedicate a few hours per week to the management of the global organization.
+We think of ourselves as one larger Global Team comprised of many smaller teams that address specific organizational needs.
+We want to build a diverse Global Team with good representation of all community members.
+
+# Timeline (approximate)
+
+-   date1-date2: Google form is open to submissions (2 weeks)
+
+-   date3-date4: Global team reviews submissions (2 weeks)
+
+-   date5-date6: Target date to notify individuals of selection.
+    (1 week)
+
+-   date7-date8: Invitation to meet with 1-3 global team members to clarify any questions or expectations to ensure a good match for all parties.
+    (2 weeks)
+
+# Selection considerations
+
+We will consider various aspects with the aim of increasing the diversity of the team.
+The following are not an exhaustive list of requirements but rather an illustration of the various paths possible.
+A new team member is not expected to satisfy all criteria.
+
+-   Geographical location
```

Comment:
```suggestion
-   Geographical location;
```

Reply:
I'm not sure the semi-colon is correct here? If anything, should it not be a normal comma?

Reply:
b/c the last bullet has many commas, i am leaning towards semi-colons